buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -10,11 +10,17 @@ plugins {
   id("otel.spotless-conventions")
 }
 
-val otelJava = extensions.create<OtelJavaExtension>("otelJava")
 
 group = "io.opentelemetry.contrib"
 
-base.archivesName.set("opentelemetry-${project.name}")
+base {
+  // May be set already by a parent project, only set if not.
+  if (!archivesName.get().startsWith("opentelemetry-")) {
+    archivesName.set("opentelemetry-$name")
+  }
+}
```

Comment:
(just in case we don't need the `compressor` parent module, we can revert this?)

Reply:
Yes